internal/librarian/command.go

```diff
@@ -239,25 +239,25 @@ func commitAndPush(ctx context.Context, r *generateRunner, commitMessage string)
 	datetimeNow := formatTimestamp(time.Now())
 	branch := fmt.Sprintf("librarian-%s", datetimeNow)
 	slog.Info("Creating branch", slog.String("branch", branch))
-	if err := r.repo.CreateBranchAndCheckout(branch); err != nil {
+	if err := repo.CreateBranchAndCheckout(branch); err != nil {
 		return err
 	}
 
 	// TODO: get correct language for message (https://github.com/googleapis/librarian/issues/885)
 	slog.Info("Committing", "message", commitMessage)
-	if err := r.repo.Commit(commitMessage); err != nil {
+	if err := repo.Commit(commitMessage); err != nil {
 		return err
 	}
 
-	if err := r.repo.Push(branch); err != nil {
+	if err := repo.Push(branch); err != nil {
 		return err
 	}
 
 	// Create a new branch, set title and message for the PR.
 	titlePrefix := "Librarian pull request"
 	title := fmt.Sprintf("%s: %s", titlePrefix, datetimeNow)
 	slog.Info("Creating pull request", slog.String("branch", branch), slog.String("title", title))
-	if _, err = r.ghClient.CreatePullRequest(ctx, gitHubRepo, branch, title, commitMessage); err != nil {
```

Comment:
It seems the logic was already there. Does this PR resolve something?

Reply:
Refactor `commitAndPush` function so that it can be called in `release init` command.

Reply:
Nice.